tests/x11/x11_login.pm

```diff
@@ -17,9 +17,9 @@ use warnings;
 use testapi;
 
 sub run {
-    type_string $username. "\n";
+    enter_cmd $username. "";
     sleep 1;
-    type_string $password. "\n";
+    enter_cmd $password. "";
```

Comment:
Here is redundand `. ""`. Wrong regexp for replacing :).

Reply:
done